plone/outputfilters/filters/example.py

```diff
@@ -18,4 +18,4 @@ def is_enabled(self):
     pattern = re.compile(r'--')
 
     def __call__(self, data):
-        return self.pattern.sub(u'—', data)
+        return self.pattern.sub(u'\xe2\x80\x94', data)
```

Comment:
@pbauer maybe you have a quicktip how to solve this doctest? py3 passes while py2 gives the following
<img width="795" alt="bildschirmfoto 2018-07-16 um 14 07 47" src="https://user-images.githubusercontent.com/511761/42757861-d23c181e-8901-11e8-8a32-182330b295e9.png">

Reply:
This is maybe due to remaining issues in PortalTransforms (https://github.com/plone/Products.CMFPlone/projects/4#card-11315725). If not you can use the Py23DocChecker-pattern to work around it.